client/js/global.js

```diff
@@ -41,6 +42,20 @@ function fmOverlay() {
   }
 
   /**
+   * Set Overlay cookies.
+   */
+  function setCookie() {
+    const name = cookieName;
+    const date = new Date();
+
+    // set cookie for 2 hours
+    date.setTime(date.getTime() + (2 * 60 * 60 * 1000));
+
+    const expires = `; expires=' + ${date.toGMTString()}`;
+    document.cookie = `${name} = ${true} ${expires}; path=/`;
```

Comment:
AFAIK, we just want to set the cookies value to true - so this from my understanding will simply evaluate `true` to `true`.  

Could probably use static text here but hey, this way later one if we want to actually use the cookie to store values we can just swap `true` for a variable.